app/models/category.rb

```diff
@@ -0,0 +1,8 @@
+class Category < ApplicationRecord
+  # belongs_to :property_category
+  # has_and_belongs_to_many :properties
```

Comment:
- [ ] Kindly delete this old piece of commented code to improve your repo's cleanliness.

Also, please remove the commented code found in:

```
- category.rb
- hosting.rb
- property_catefory.rb
- categories_controller.rb
- hostings_controller.rb
- properties_controller.rb
```

Reply:
Thank you @Meltrust . The commented code was already eliminated.